package/utils/uencrypt/src/CMakeLists.txt

```diff
@@ -5,16 +5,27 @@ cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 project(uencrypt LANGUAGES C)
 
 option(USE_WOLFSSL "Use WolfSSL as crypto provider" OFF)
-if (USE_WOLFSSL)
-	add_definitions(-DUSE_WOLFSSL)
-	find_library(WOLFSSL_LIBRARY wolfssl REQUIRED)
-	set(CRYPTO_LIBRARIES ${WOLFSSL_LIBRARY})
+option(USE_MBEDTLS "Use mbedTLS as crypto provider" OFF)
+if (USE_MBEDTLS)
+	if (USE_WOLFSSL)
+		message(WARNING "USE_MBEDTLS and USE_WOLFSSL are bot set. Building with USE_MBEDTLS.")
```

Comment:
This warning has a typo: ` ... are both set ...`

Reply:
Thanks for spotting this.